groups/groups.go

```diff
@@ -406,10 +406,24 @@ func buildActionableInput(r *http.Request) (*ActionableInput, error) {
 		return nil, err
 	}
 
+	err = input.Validate()
+	if err != nil {
+		return nil, err
+	}
+
 	return input, nil
 }
 
-func decodeResponseBodyAndValidate(body []byte) (*ServiceGroup, error) {
+func (i *ActionableInput) Validate() error {
```

Comment:
Read out of context I feel like the name `ActionableInput` loses meaning. `RequestParams` or `RequestInput` might be more sufficient.

Reply:
Ok, we can change that for sure :)